api/RTCStatsReport.json

```
@@ -255,9 +258,8606 @@
+                "version_added": "106"
+              },
+              "firefox_android": "mirror",
+              "ie": {
+                "version_added": false
+              },
+              "oculus": "mirror",
+              "opera": "mirror",
+              "opera_android": "mirror",
+              "safari": {
+                "version_added": "13.1> ≤14.1"
+              },
+              "safari_ios": {
+                "version_added": "≤16.4"
+              },
+              "samsunginternet_android": "mirror",
+              "webview_android": "mirror"
+            },
+            "status": {
+              "experimental": false,
+              "standard_track": true,
+              "deprecated": false
+            }
+          }
+        },
+        "jitter": {
+          "__compat": {
+            "description": "<code>jitter</code> in 'inbound-rtp' stats",
+            "spec_url": "https://w3c.github.io/webrtc-stats/#dom-rtcreceivedrtpstreamstats-jitter",
+            "support": {
+              "chrome": {
+                "version_added": "≤80"
+              },
+              "chrome_android": {
+                "version_added": "≤112"
+              },
+              "edge": {
+                "version_added": "≤80"
+              },
+              "firefox": {
+                "version_added": "27"
+              },
+              "firefox_android": "mirror",
+              "ie": {
+                "version_added": false
+              },
+              "oculus": "mirror",
+              "opera": "mirror",
+              "opera_android": "mirror",
+              "safari": {
+                "version_added": "≤13.1"
+              },
+              "safari_ios": {
+                "version_added": "≤16.4"
+              },
+              "samsunginternet_android": {
+                "version_added": "≤20.0"
+              },
+              "webview_android": {
+                "version_added": "≤112"
+              }
+            },
+            "status": {
+              "experimental": false,
+              "standard_track": true,
+              "deprecated": false
+            }
+          }
+        },
+        "jitterBufferDelay": {
+          "__compat": {
+            "description": "<code>jitterBufferDelay</code> in 'inbound-rtp' stats",
+            "spec_url": "https://w3c.github.io/webrtc-stats/#dom-rtcinboundrtpstreamstats-jitterbufferdelay",
+            "support": {
+              "chrome": {
+                "version_added": "86"
+              },
+              "chrome_android": "mirror",
+              "edge": "mirror",
+              "firefox": {
+                "version_added": "104"
+              },
+              "firefox_android": "mirror",
+              "ie": {
+                "version_added": false
+              },
+              "oculus": "mirror",
+              "opera": "mirror",
+              "opera_android": "mirror",
+              "safari": {
+                "version_added": "13.1> ≤14.1"
```

This indicates "not in safari 13" = https://bugs.webkit.org/show_bug.cgi?id=212668. So perhaps we can specify 14.1 here?

```suggestion
                "version_added": "14.1"
```